### ISSUE TITLE
Premise for this game

### ISSUE THREAD

**Issue by andr7430** [SyndicateFighter/SyndicateFighter#2]
We need a story line for this game. 
My pitch:
Assemble the best business team in the world to become the best business in the world. Taking down all the big corporations in the world!

You start off with your basic classes:

Technology
Management
Customer Service (<- dis us btw)

Then you build your team from there. Collecting other classes and stuff.

**Comment by nohe427** [comment]
We could call it Trump quest.  You start as a small boy with a tiny million dollar loan from your father and then go out to own all the businesses in New York City and eventually the white house. :wink:

**Comment by andr7430** [comment]
I was thinking the direction we are headed is:
Beat your workers to a pulp first then try to hire them.... Is that the message we want to convey? Can we just offer them a contract outright?

**Comment by nohe427** [comment]
Good point. We can do that but I was thinking rather than HP call it something else and negotiate with them until we can get them to a certain price point.  What do you think of that?

**Comment by andr7430** [comment]
hmm but what about when fighting gyms/corporations?

**Comment by andr7430** [comment]
potentially could learn from:
https://play.google.com/store/apps/details?id=net.kairosoft.android.gamedev3en
http://www.androidshock.com/category/strategy/business-simulation/

**Comment by nohe427** [comment]
I think we could do the same thing but at the end of a corporation fight you acquire all the children companies.  And the trainer battles are other children companies of the parent you are training to fight.

**Comment by andr7430** [comment]
so every time you fight a battle, you acquire all their workers?

**Comment by nohe427** [comment]
Nah, they become a child corporation that you can visit to train certain characters at. Kind of like a minigame. (future releases).  The more child corporations you own, the more higher level employees you can retain because they want to work for the big boss.  You at the very least, win money and daily in game income

**Comment by andr7430** [comment]
I see what you are getting at with more child corporations = higher level employees. Defeating corporation = new child corporation. 
I am not sure when you mean by the trainer battles, and battle mechanics though...

**Comment by nohe427** [comment]
Trainer battles are those random battles you encounter on the way to the gym with other trainers

**Comment by andr7430** [comment]
Yea, but what happens when you fight them? Do you acquire them somehow?

**Comment by nohe427** [comment]
Nah, you acquire capital(cash) from them

**Comment by andr7430** [comment]
gotcah! Also need to determine how corporation battles / gym battle happens.

**Comment by nohe427** [comment]
Maybe after you defeat a set number of their child corporations

**Comment by nohe427** [comment]
We could rate limit it for trainer battles towards one a day

**Comment by andr7430** [comment]
I was thinking since we can control the geotriggers, and giant corporation battles happen every Fri-Sun at city center ie dt redlands?

**Comment by nohe427** [comment]
Yeah but sometimes city centers don't work i.e., Baltimore

**Comment by andr7430** [comment]
as for trainer battles, also controlled by geotriggers, but they only happen once an hour at specific locations?

**Comment by andr7430** [comment]
a really nice park in Baltimore, or do you guys not have those?

**Comment by nohe427** [comment]
Trainer battles once an hour and you can travel to their location?  I like the idea.  After what recently happened in Baltimore, I would not want to travel with a smartphone

**Comment by nohe427** [comment]
Maybe we could spread out the gym battles to every 5 miles and the trainer miles to every 2 miles?

**Comment by andr7430** [comment]
what about obtaining new workers? every mile?

**Comment by nohe427** [comment]
And do a 500 M buffer around the geo trigger location to initiate the battle?

Obtaining new workers could be every mile or just random events?

**Comment by andr7430** [comment]
how about trainer battles to be random events ~1hr each.

**Comment by nohe427** [comment]
That works.  And new worker interactions are through parks or shopping centers?

**Comment by andr7430** [comment]
yea we set geotrigger in parks and shopping centers. or every 2 miles.

**Comment by andr7430** [comment]
each national / state park get their own special rare worker?

**Comment by nohe427** [comment]
That would be awesome!!!

**Comment by nohe427** [comment]
If we do offline geotriggers, would we need to download the areas onto the device to activate the triggers?  I assume a national area would be a very large file

**Comment by andr7430** [comment]
was looking into the doc for geotriggers. I am not sure if i can limit the number of triggers I Dl....
might have to go with the online approach.

**Comment by nohe427** [comment]
Limit it as in every hour a trigger can be activated or ....

**Comment by nohe427** [comment]
Starter workers should be a temp

**Comment by andr7430** [comment]
Limit as in limit the # of triggers that you dl to your device.
